include/functions.h

```diff
@@ -2534,12 +2534,12 @@ void func_80140E80(void* param_1);
 // void func_8014151C(UNK_TYPE1 param_1, UNK_TYPE1 param_2, UNK_TYPE1 param_3, UNK_TYPE1 param_4, UNK_TYPE4 param_5, UNK_TYPE4 param_6);
 // void func_80141678(UNK_TYPE1 param_1, UNK_TYPE1 param_2, UNK_TYPE1 param_3, UNK_TYPE1 param_4, UNK_TYPE4 param_5);
 // void func_80141778(void);
-void func_801418B0(void* arg0);
-void func_80141900(void* arg0);
-// void func_80141924(void);
-// void func_80141C34(void);
-void VisMono_Draw(void* arg0, Gfx** gfx, u32 arg2);
-// void func_8014204C(void);
+void VisMono_Init(VisMono* this);
+void VisMono_Destroy(VisMono* this);
+// void VisMono_DesaturateTLUT(void);
+// void VisMono_Init(void);
```

Comment:
This seems to be a duplicate? Or something got caught in the crossfire, anyway.

Reply:
Yeah, I have no idea what happened here. I fixed it up, and also added the correct parameters to the function declarations.